lua/weapons/grav_blaster.lua

```diff
@@ -0,0 +1,117 @@
+if SERVER then
+	resource.AddSingleFile("materials/entities/grav_blaster.png")
+	AddCSLuaFile()
+end
+
+SWEP.PrintName = "Grav Blaster"
+SWEP.Category = TASWeapons.Category
+SWEP.Author = "yogwoggf"
+SWEP.Instructions = [[Left click to shoot a burst of gravity which knocks away other players and props.
+Uses combine ball ammunition.
+Strat: Jump and shoot at the ground to propel yourself into the air. You will be hurt though!]]
+
+SWEP.Spawnable = true
+
+SWEP.Primary.ClipSize = 8
+SWEP.Primary.DefaultClip = 8
+SWEP.Primary.Automatic = true
+SWEP.Primary.Ammo = "AR2AltFire"
+
+SWEP.Secondary.ClipSize = -1
+SWEP.Secondary.DefaultClip = -1
+SWEP.Secondary.Automatic = true
+SWEP.Secondary.Ammo = "none"
+
+SWEP.Weight = 5
+SWEP.AutoSwitchTo = false
+SWEP.AutoSwitchFrom = false
+
+SWEP.Slot = 2
+SWEP.SlotPos = 1
+
+SWEP.DrawAmmo = true
+SWEP.DrawCrosshair = true
+SWEP.UseHands = true
+SWEP.ViewModelFOV = 50
+
+SWEP.ViewModel = Model("models/weapons/c_physcannon.mdl")
+SWEP.WorldModel = Model("models/weapons/w_physics.mdl")
+
+SWEP.ShootSound = Sound("weapons/physcannon/energy_disintegrate4.wav")
+SWEP.LoadSound = Sound("weapons/physcannon/physcannon_charge.wav")
+-- Sound actually ends at 1.5 seconds but has an extra second of nothing.
+-- Also, SoundDuration is known to be unstable on Linux.
+SWEP.LoadSoundDurationSecs = 1.5
+
+SWEP.PrimaryRateOfFire = 1.5 -- Per second
```

Comment:
Given this is a custom property, why are you giving it a non-self documenting name in the first place?

Also can we please not put custom properties in the `SWEP` struct, it makes it non-obvious what actually gets used by gmod and what's a constant we use later in the code